elasticsearch/pom.xml

```diff
@@ -234,7 +234,6 @@
                                 
                                 <!-- HSEARCH-2393 Remove depencies to Lucene in tests for features common to Lucene and Elasticsearch -->
                                 <exclude>**/ProgrammaticMappingTest.java</exclude><!-- Analyzers -->
-                                <exclude>**/ProjectionQueryTest.java</exclude><!-- Projection constant __HSearch_Document -->
```

Comment:
Not related to this line but to the commit message of 78a0f9add77d9180aacd309f3c737237a1074f64: look for "HSEARCH-HSEARCH"